examples/node/lib/rest/api/v2010/account.ts

```diff
@@ -19,6 +19,7 @@ import V2010 from "../V2010";
 const deserialize = require("../../../base/deserialize");
 const serialize = require("../../../base/serialize");
 import { CallListInstance } from "./account/call";
+import PhoneNumberCapabilities from "../../../interfaces";
```

Comment:
Where do these get declared?

Reply:
PhoneNumberCapabilities is a custom type in interfaces in twilio-node. This is needed to declare the type of [test_object](https://github.com/twilio/twilio-oai-generator/blob/356c8f1defb593a6eef664cafd3a246b95c97e50/examples/node/lib/rest/api/v2010/account.ts#L273)